plugins/modules/managed_serviceaccount_rbac.py

```diff
@@ -33,8 +33,12 @@
         type: str
         required: True
     rbac_template:
-        description: Path to the RBAC role/clusterrrole/rolebinding/clusterrolebinding YAML file or a directory of YAML files.
-        type: str
+        description:
+        - Path to the file or directory that contains the role/clusterrrole/rolebinding/clusterrolebinding configuration.
+        - The path specified should either be the absolute or relative to the location of the playbook.
+        - In order to avoid potential resource name collision, the name specified in the RBAC files
+          will be appended with the last 12 digit of UID of the target managed-serviceaccount.
```

Comment:
"... will be appended to with the last 12 digits of the target managed-serviceaccount's UID."

^ Maybe better wording?

Reply:
"In order to avoid potential resource name collision, the last 12 digits of the target managed-serviceaccount's UID will be appended to the name specified in the RBAC files."

Also, I see we flip back and forth between `ManagedServiceAccount` and `managed-serviceaccount` in the docs ¯\_(ツ)_/¯